shell/client/admin/user-details-client.js

```diff
@@ -230,4 +300,67 @@ Template.newAdminUserDetails.events({
       isAdmin: false,
     });
   },
+
+  "click [name=\"delete-account\"]"(evt, instance) {
```

Comment:
This is a pattern I've been using recently, and I think it's wise to separate the concerns of event routing from CSS style.

In particular, it is annoying that when attempting to refactor or rework CSS, you must also consider the fact that event handlers have bound to that class name.  So you can't trivially merge things, or reuse classes, because you might change the behavior of event handlers elsewhere.

Having events dispatch based on name instead of class makes this less error-prone and less cognitive load when refactoring.

Meteor recommends that you use `.js-` selectors: https://guide.meteor.com/blaze.html#js-selectors-for-events ; while I produced the `name=` approach before reading that, the effective result seems similar.

(All that said, in almost all cases you want to have a unique function called for events on a unique DOM node, so having to make events dispatch via additional selectors that only exist for event dispatch, rather than placing them on the DOM nodes directly, is kinda sad.  This is another thing that React gets remarkably right.)